ethportal-api/src/lib.rs

```diff
@@ -5,6 +5,7 @@
 mod discv5;
 mod history;
 pub mod types;
+mod utils;
```

Comment:
What is your opinion about exporting this module or parts of it as public?

Reply:
I was on the fence but now favour `pub mod utils`. On the one hand exposing utils increases the scope of the crate's contract. On the other, it reduces friction for the `hex_*` methods, which is desirable